app/presenters/statistics_filter_json_presenter.rb

```diff
@@ -1,4 +1,9 @@
 class StatisticsFilterJsonPresenter < DocumentFilterPresenter
+  def as_json(options = nil)
+    super.merge atom_feed_url: context.filter_atom_feed_url,
```

Comment:
I realise this is copied from the other classes so I'm not too fussed, but I would extract `atom_feed_url` and `email_signup_url` into separate methods on this class, and then in this method do something like `super.merge(atom_feed_url: atom_feed_url, email_signup_url: email_signup_url)`.

Reply:
Should I do the same in `announcement_filter_json_presenter` and `publication_filter_json_presenter` while I'm at it?

Reply:
Yeah, that sounds good to me!